src/dataServices/apiDataService.ts

```diff
@@ -12,21 +12,23 @@ import {
     UpdateResourceRequest,
     vReadResourceRequest,
 } from 'fhir-works-on-aws-interface';
-import axios from 'axios';
+import axios, { AxiosInstance } from 'axios';
+import IamAuth from '../auth/iamAuth';
 
 export class ApiDataService implements Persistence {
     updateCreateSupported: boolean = false;
 
-    private INTEGRATION_TRANSFORM_URL: string;
+    private axiosInstance: AxiosInstance;
 
-    constructor(integrationTransformUrl: string) {
-        this.INTEGRATION_TRANSFORM_URL = integrationTransformUrl;
+    constructor(integrationTransformUrl: string, axiosInstance: AxiosInstance | undefined = undefined) {
```

Comment:
```suggestion
    constructor(integrationTransformUrl: string, axiosInstance?: AxiosInstance = undefined) {
```
double check before merging, but I think these are equivalent

Reply:
that worked.